docs/creating-manifest/events.md

```diff
@@ -1757,7 +1757,109 @@ onAfterRemoveSSL:
 ```
 @@!
 
+### onBeforeInstallAddon  
+The event is executed before add-on installation  
+
+Event Placeholders:  
+
+-   `${event.params.}`:  
+    - `settings` - ***What is it***  
```

Comment:
values of http://docs.cloudscripting.com/creating-manifest/visual-settings/

Reply:
- `settings` - custom form with predefined user input elements

Reply:
done